### PR TITLE
Update SRTS-A01.md

### PR DESCRIPTION
**Pull request by HolgerAusB**
Added message to calibrate valve

As suggested by @KF2806 in [#1748 ](https://github.com/Koenkk/zigbee2mqtt/discussions/17481#discussioncomment-7297378) and confirmed by @Ra72xx and myself with Aqara E1 TRV (aka Xiaomi SRTS-A01)

### CODE REVIEW

**Comment by Koenkk** [comment]
Thanks!

File: docs/devices/SRTS-A01.md [hunk 1]
```diff
@@ -65,9 +65,11 @@ The possible values are: `internal`, `external`.
 ### Calibrated (binary)
 Is the valve calibrated.
 Value can be found in the published state on the `calibrated` property.
-It's not possible to read (`/get`) or write (`/set`) this value.
```
Comment (Koenkk) [inline docs/devices/SRTS-A01.md:68]: @HolgerAusB I noticed now you removed `or write (`/set`)`, but AFAIK this is valid (it is not possible to set it)

Comment (HolgerAusB) [reply docs/devices/SRTS-A01.md:68]: @Koenkk I am not perfect in English. Maybe the wording is not right. You can't `/set/calibrated`, correct. But I could `/set/calibrate` to start the calibration. If that makes a difference, please change the sentence as needed.

Comment (Koenkk) [reply docs/devices/SRTS-A01.md:68]: The `calibrate` expose was missing, I added it in https://github.com/Koenkk/zigbee-herdsman-converters/commit/2a58fbef89b13d28334c49ef6fa60f4562646b64

**Pull request merged**